code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm

```diff
@@ -0,0 +1,193 @@
+	return
+
+//////////////
+///Subtypes///
+//////////////
+
+/datum/eldritch_knowledge/spell
+	var/obj/effect/proc_holder/spell/spell_to_add
+
+/datum/eldritch_knowledge/spell/on_gain(mob/user)
+	var/obj/effect/proc_holder/S = new spell_to_add
+	user.mind.AddSpell(S)
+	. = ..()
+
+/datum/eldritch_knowledge/spell/on_lose(mob/user)
+	user.mind.RemoveSpell(spell_to_add)
+	. = ..()
+
+/datum/eldritch_knowledge/curse
+	var/timer = 5 MINUTES
+	var/list/fingerprints = list()
+
+/datum/eldritch_knowledge/curse/recipe_snowflake_check(list/atoms, loc)
+	fingerprints = list()
+	for(var/X in atoms)
+		var/atom/A = X
+		fingerprints |= A.return_fingerprints()
+	listclearnulls(fingerprints)
+	if(LAZYLEN(fingerprints) == 0)
+		return FALSE
+	return TRUE
+
+/datum/eldritch_knowledge/curse/on_finished_recipe(mob/living/user,list/atoms,loc)
+
+	var/list/compiled_list = list()
+
+	for(var/mob/living/carbon/human/H in GLOB.alive_mob_list)
+		if(fingerprints[md5(H.dna.uni_identity)])
+			compiled_list |= H.real_name
+			compiled_list[H.real_name] = H
+
+	if(LAZYLEN(compiled_list) == 0)
+		to_chat(user, "<span class='warning'>The items don't posses required fingerprints.</span>")
+		return
+
+	var/chosen_mob = input("Select the person you wish to curse","Your target") as null|anything in sortList(compiled_list, /proc/cmp_typepaths_asc)
+	if(!chosen_mob)
+		return ..()
+	curse(compiled_list[chosen_mob])
+	addtimer(CALLBACK(src, .proc/uncurse, compiled_list[chosen_mob]),timer)
+	. = ..()
+
+/datum/eldritch_knowledge/curse/proc/curse(mob/living/chosen_mob)
+	return
+
+/datum/eldritch_knowledge/curse/proc/uncurse(mob/living/chosen_mob)
+	if(!chosen_mob)
+		return
+
+/datum/eldritch_knowledge/summon
+	var/mob/living/mob_to_summon
+
+
+/datum/eldritch_knowledge/summon/on_finished_recipe(mob/living/user,list/atoms,loc)
+	. = ..()
+	var/mob/living/summoned = new mob_to_summon(loc)
+	message_admins("[summoned.name] is being summoned by [user.real_name] in [loc]")
+	var/list/mob/dead/observer/candidates = pollCandidatesForMob("Do you want to play as a [summoned.real_name]", ROLE_ECULTIST, null, ROLE_ECULTIST, 50,summoned)
+	if(!LAZYLEN(candidates))
+		return
+	var/mob/dead/observer/C = pick(candidates)
+	message_admins("[key_name_admin(C)] has taken control of ([key_name_admin(summoned)]).")
+	summoned.ghostize(0)
+	summoned.key = C.key
+
+	to_chat(summoned,"<span class='warning'>You are bound to [user.real_name]'s' will! Don't let your master die, protect him at all cost!</span>")
+///////////////
+///Base lore///
+///////////////
+
+/datum/eldritch_knowledge/spell/basic
+	name = "Break of dawn"
+	desc = "Starts your journey in the mansus. Allows you to transmute a soul bottle using a glass shard and a broken drinking bottle. Use a soul bottle on a dead person to harvest their soul. Use it then on ancient lore to  gain a charge. You can additionally harvest pierced realities. It takes a minute to harvest them and after you do so they are visible to everyone."
```

Comment:
*10 seconds

Reply:
oh shit i didnt update the description, i will do that tommorow